src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/CurrentPayment.cs

```diff
@@ -0,0 +1,61 @@
+﻿using SwedbankPay.Sdk.PaymentInstruments;
+using System;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    internal class CurrentPayment : Identifiable, ICurrentPayment
+    {
+        public CurrentPayment(PaymentOrderPaymentDto payment)
+            : base(new Uri(payment.Id, UriKind.RelativeOrAbsolute))

```

Comment:
`Identifiable.ctor(string id)` exists, no?

```suggestion
            : base(payment.Id)
```

Reply:
That was created after this code was written, but yes.